crates/proof-of-sql/sol_src/tests/TestYulImport.t.psol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+library TestScript {
+    function testWeCanImportYulFromAnotherFile() public pure {
+        bytes memory point0 = hex"0000000000000000" hex"0000000000000000" hex"0000000000000000"
+            hex"0000000000000002" hex"0000000000000000" hex"0000000000000000" hex"0000000000000000"
+            hex"0000000000000005";
+        uint256 length0 = 1;
+        uint256 numVars0 = 2;
+        uint256 modulus0 = 10007;
+        uint256 result0;
+        // solhint-disable-next-line no-inline-assembly
+        assembly {
+            // IMPORT-YUL ../base/LagrangeBasisEvaluation.sol:compute_truncated_lagrange_basis_sum
+            // solhint-disable-next-line no-empty-blocks
+            function compute_truncated_lagrange_basis_sum(length, point, num_vars, modulus) -> result {}
```

Comment:
this line isn't needed right?

Reply:
Correct. It's needed for the sake of the linter, so it knows what the function signature is.